test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
Part of the reason not to run the kubernetes e2e tests is time. We aren't trying to validate the entirety of a kube install just that when we provision an acs cluster that it comes up in a valid state. The e2e tests for kubernetes should be validating that kubernetes itself works on azure which is an entirely different set of tests. I think we should move forward with porting our validation tests to ginkgo.